SwiftUiMovieDb/Protocol/DescriptiveErrorType.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  DescriptiveErrorType.swift
+//  SwiftUiMovieDb
+//
+//  Created by mac on 04/11/23.
+//
+
+import Foundation
+
+struct MDBErrorConstants {
```

Comment:
This struct is not needed let's create localization file and move all the strings in it and directly use it in below enum.